### DISCUSSION TITLE
[Feature] Add single picture download button

### DISCUSSION THREAD

**Discussion by fwsmit** [immich-app/immich#8414] [Feature Request]
### The feature

Currently, if you get a link to a shared album and want to download a picture you have to long press it before you can download it. This is very hard to find for a first time user. The button to download all pictures is easy to find, but not practical if there are lots of pictures.
I would suggest adding a download button in the single picture view. This would make it easier to download single pictures from the web.

Note that this button already exists when logged in, but it's buried in the hamburger menu. But I'm trying to improve the experience for first time users. 

![Screenshot_20240331_194807_com android chrome](https://github.com/immich-app/immich/assets/23078054/d3cc7e33-0a80-4a4f-a6c9-1738fbfdd4b2)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This is now implemented

**Reply by fwsmit** [reply]
Nice, thank you!
